### PR TITLE
Add `/cookies` endpoint to smoke test express server

### PR DESCRIPTION
**Pull request by filfreire**
This PR adds a simple endpoint to the smoke tests' express server.

Given a `curl` like:
```bash
curl --request GET \
  --url http://localhost:4010/cookies \
  --cookie 'foo=bar; insomnia-test-cookie=value123'
```

It'll print in plaintext both the cookie sent by the user. It'll also reply back as well with a default response cookie (`insomnia-test-cookie=value123`).

Example when used in Insomnia:

Screenshot 2022-01-27 at 18 12 33

~~This serves primarily for now to aid in some tests for Insomnia. Later on we can reuse the endpoint or adapt it for proper playwright tests.~~

### Update

- Updated the fixture yaml to include the endpoint
- Added steps to the smoke checks to show this endpoint works (and indirectly that we can resend a cookie we receive as response) (follow up to @jackkav's [comment](https://github.com/Kong/insomnia/pull/4419#issuecomment-1025151160))

> Related issue: https://linear.app/insomnia/issue/INS-1371/investigate-cookies-and-add-e2e-test

### CODE REVIEW

**Review by wongstein — APPROVED** [review]

**Comment by jackkav** [comment]
Would be great to see e2e covering the cookie feature to show how this endpoint would work.
For the cancel test I just edited the fixture yaml by hand. Not sure if you can do the same with this one.

File: packages/insomnia-smoke-test/tests/app.test.ts [hunk 1]
```diff
@@ -38,4 +38,12 @@ test('can send requests', async ({ app, page }) => {
   await page.click('button:has-text("GETsends request with basic authentication")');
   await page.click('text=http://127.0.0.1:4010/auth/basicSend >> button');
   await page.click('text=200 OK');
+
+  // Send request, check if no cookie was sent (server will reply with received cookies + a new cookie as response)
+  await page.click('button:has-text("GETsends request with cookie and get cookie in response")');
+  await page.click('text=http://127.0.0.1:4010/cookiesSend >> button');
+  await page.click('text=undefined',);
```
Comment (gatzjames) [inline packages/insomnia-smoke-test/tests/app.test.ts]:
```suggestion
  await page.click('text=undefined');
```
😅

Comment (gatzjames) [reply packages/insomnia-smoke-test/tests/app.test.ts]: Checked this and looks fine.
We could make the cookie endpoint set the cookies we send and check the headers instead if we want to test setting/getting many cookies in the future

**Review by gatzjames — APPROVED** [review]
Added a comment for how the /cookie endpoint returns results but we can improve it later once we start using it in more complex scenarios.
🚀 💯

**Pull request merged**